### ISSUE TITLE
Can't run because no spec files were found. use docker-compose.yml

### ISSUE THREAD

**Issue by HikaruTakemoto** [cypress-io/cypress#23529]
### Current behavior

docker-compose.yml file
```
version: '3'
services:
  e2e-chrome:
    image: 'cypress/included:10.6.0'
    shm_size: 2g
    # container_name: cypress
    # "cypress/included" images have entrypoint set to globally installed cypress
    # so the command can simply add any arguments
    command: '--browser chrome --spec cypress/e2e/login.cy.ts'
    volumes:
      - ./cypress:/cypress
      - ./node_modules:/node_modules
      - ./cypress.env.json:/cypress.env.json
      - ./package.json:/package.json
      - ./tsconfig.json:/tsconfig.json
    environment:
      TZ: Asia/Tokyo
```

I use `docker compose up` and I get following message.

```
e2e-chrome_1  | Can't run because no spec files were found.
e2e-chrome_1  | 
e2e-chrome_1  | We searched for specs matching this glob pattern:
e2e-chrome_1  | 
e2e-chrome_1  |   > /cypress/e2e/login.cy.ts
```

When I don't use `--spec cypress/e2e/login.cy.ts`, the test runs.

I have `login.cy.ts` file
```
pwd
~~/e2e-cypress
ls cypress/e2e/login.cy.ts 
cypress/e2e/login.cy.ts 
```

### Desired behavior

E2Etest runs.

### Test code to reproduce

It doesn't matter what's in the test.

### Cypress Version

10.6.0

### Node version

16.5.0

### Operating System

macOS 11.6

**Comment by HikaruTakemoto** [comment]
when I use `cypress/included:10.3.0`, I can use `--spec cypress/e2e/login.cy.ts`

**Comment by tbiethman** [comment]
Hi @HikaruTakemoto, thanks for the report. I believe this to be the same issue as https://github.com/cypress-io/cypress/issues/23380, which should be fixed for the 10.7.0 release. I will close this issue as a duplicate; please follow along with https://github.com/cypress-io/cypress/issues/23380 for more info.
